lib/littlefs/README.md

```diff
@@ -0,0 +1,7 @@
+littlefs library
+================
+
+The upstream source for the files in this directory is
+https://github.com/littlefs-project/littlefs
+
+The current files come from v2.8.2
```

Comment:
_Maybe_ worth bumping (after testing, of course) to the latest release? https://github.com/littlefs-project/littlefs/releases/tag/v2.11.1